packages/strapi-design-system/src/DatePicker/DatePicker.stories.mdx

```diff
@@ -0,0 +1,41 @@
+<!--- DatePicker.stories.mdx --->
+
+import { useState } from 'react';
+import { Meta, Story, Canvas } from '@storybook/addon-docs/blocks';
+import { DatePicker } from './DatePicker';
+
+<Meta
+  title="DatePicker"
+  component={DatePicker}
+  parameters={{
+    design: {
+      type: 'figma',
+      url: 'TODO: Fill the according figma file',
+    },
+  }}
+/>
+
+# DatePicker
+
+This is the doc of the `DatePicker` component
+
+## DatePicker
+
+Description...
+
+<Canvas>
+  <Story name="base">
+    {() => {
+      const [date, setDate] = useState();
+      return (
+        <DatePicker
+          onChange={setDate}
+          selectedDate={date}
+          label="Date picker"
+          name="datepicker"
+          selectedDateLabel={(formattedDate) => `Date picker, current is ${formattedDate}`}
+        />
```

Comment:
I tried to add another Datepicker to test the behaviour of two date pickers and there is a little bug on the popover. The first time you click on the first date picker, the popover is displayed below the second one.

![datepicker](https://user-images.githubusercontent.com/7756284/120185778-ef01bb00-c212-11eb-8d5f-c9e9b76e025d.gif)

Reply:
ooh wooown that's weird 🤔

Reply:
This helped makes the code of popover way easier, thank you 😊